app/assets/javascripts/discourse/app/widgets/post-menu.js

```diff
@@ -477,27 +482,65 @@ export default createWidget("post-menu", {
       collapsed: true,
       likedUsers: [],
       readers: [],
-      adminVisible: false,
     };
   },
 
   buildKey: (attrs) => `post-menu-${attrs.id}`,
 
   attachButton(name) {
-    let buttonAtts = buildButton(name, this);
+    let buttonAttrs = buildButton(name, this);
+
+    if (buttonAttrs?.component) {
+      return [
+        new RenderGlimmer(
```

Comment:
Just so I understand, this is a new way of rendering a dynamic component as a replacement for `{{component this.componentNameString}}`?

Reply:
No it's a way to render a glimmer component in a widget